docker_run.py

```diff
@@ -31,7 +31,7 @@
 
     cmd += " -e DISPLAY -e QT_X11_NO_MITSHM=1 -v /tmp/.X11-unix:/tmp/.X11-unix:rw "     # enable graphics
     cmd += " -v ~/.ssh:%(home_directory)s/.ssh " % {'home_directory': home_directory}   # mount ssh keys
-    cmd += " -v %(source_dir)s:/6-881-examples/ " % {'source_dir': source_dir}
+    cmd += " -v '%(source_dir)s':/6-881-examples/ " % {'source_dir': source_dir}
```

Comment:
Is this change necessary? This script has been used by Greg in a few of his repos for a while.

Reply:
It enables me to run this script on my laptop. Because my repo is in a Dropbox folder with the forced name of “Dropbox (MIT)”, without the quotes it adds a space in the command itself. This throws an error when trying to execute the command in the terminal which says something like “ ( not recognized”

Reply:
It enables me to run this script on my laptop. Because my repo is in a Dropbox folder with the forced name of “Dropbox (MIT)”, without the quotes it adds a space in the command itself. This throws an error when trying to execute the command in the terminal which says something like “ ( not recognized”